gcp/appengine/blog/content/posts/introducing-broad-c-c++-support/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Introducing broad C/C++ vulnerability management support"
+date: 2023-10-31T01:00:00Z
+draft: true
+author: Andrew Pollock and Oliver Chang
+---
+OSV is committed to bringing our users comprehensive, accurate and timely open source vulnerability information. Over the last year, we’ve released a number of new features in pursuit of this goal including:
+- [OSV-Scanner’s call graph analysis for Go and Rust](https://google.github.io/osv-scanner/experimental/#scanning-with-call-analysis)
+- Adding six new ecosystems to the database
+- [The determineversion API](https://osv.dev/blog/posts/using-the-determineversion-api/), which expanded access to C/C++ vulnerabilities for OSS-Fuzz projects 
+
+Today we are announcing that OSV advisories now include vulnerable commit ranges. **Vulnerable commit ranges, along with the previously announced experimental determineversion API, will enable vulnerability management for software with C and C++ dependencies, which has been one of the last gaps in coverage in OSV.dev’s database.**
+<!--more-->
+
+Because the C/C++ ecosystem doesn’t have a centralized package register, source code identifiers (e.g. git hashes) are the best way to identify C/C++ libraries. Typically vulnerabilities are associated with versions, not git hashes, making C/C++ vulnerability matching difficult. The new commit level vulnerability information will allow users to confidently match their dependencies to known vulnerabilities within the OSV database. 
+
+Vulnerable commit ranges provide a granular approach that more closely follows the development process, matching vulnerabilities to a range of upstream commits instead of matching vulnerabilities to a package version. This precise information allows for accurate identification of git branches cut from known vulnerable commits and leads to more accurate vulnerability information.
```

Comment:
@andrewpollock you are right, it is a proper noun. I think I hadn't considered it because usually when you are using git, you are doing so in command line where lower case is normal. But yes, I agree.